pillar/fluentd/ocw_cms.sls

```diff
@@ -0,0 +1,75 @@
+{% from "fluentd/record_tagging.jinja" import record_tagging with context %}
+
+fluentd:
+  plugins:
+    - fluent-plugin-secure-forward
+  configs:
+    - name: ocwcms
+      settings:
+        - directive: source
+          attrs:
+            - '@id: ocwcms_apache_access_log'
+            - '@type': tail
+            - enable_watch_timer: 'false'
+            - tag: ocwcms.apache.access
+            - path: /var/log/apache2/access.log
+            - pos_file: /var/log/apache2/access.log.pos
+            - nested_directives:
+              - directive: parse
+                attrs:
+                  - '@type': apache2
+                  - keep_time_key: 'true'
+        - directive: source
+          attrs:
+            - '@id: ocwcms_apache_error_log'
+            - '@type':  tail
+            - enable_watch_timer: 'false'
+            - tag: ocwcms.apache.error
+            - path: /var/log/apache2/error.log
+            - pos_file: /var/log/apache2/error.log.pos
+            - nested_directives:
+              - directive: parse
+                attrs:
+                  - '@type': apache_error
+        - directive: source
+          attrs:
+            - '@id: ocwcms_zope_event_log'
+            - '@type': tail
+            - enable_watch_timer: 'false'
+            - tag: ocwcms.zope.event
+            - path: /usr/local/Plone/zeocluster/var/client*/event.log
+            - pos_file: /usr/local/Plone/zeocluster/var/client_event.log.pos
```

Comment:
If we're using path globbing for the file, then I'm not sure if it would work to have a single pos_file defined...

Reply:
I was following https://docs.fluentd.org/v1.0/articles/in_tail#pos_file-(highly-recommended) , which says

> pos_file handles multiple positions in one file so no need multiple pos_file parameters per source

Reply:
Good to know :+1: I'm glad you did your homework :smile:

Reply:
:-)

As above, the 0.12 version of the doc says that one pos file can store multiple positions.